haystack/components/generators/hugging_face_local.py

```diff
@@ -51,6 +52,9 @@ def __init__(
         stop_words: Optional[List[str]] = None,
     ):
         """
+        Creates an instance of a HuggingFaceLocalGenerator. Unless specified otherwise in the `model`, the instance will
+        be initialized with the `google/flan-t5-base` model.
+
         :param model: The name or path of a Hugging Face model for text generation,
             for example, "google/flan-t5-large".
             If the model is also specified in the `huggingface_pipeline_kwargs`, this parameter will be ignored.
```

Comment:
Adding this here as the affected line (lines 75 and 76) are not changed in this PR - Let's not use bare URLs in the docstring. Markdown links should work.